Readme.md

```diff
@@ -38,14 +38,16 @@ It's recommended to add `./tests/Doc` to your `.gitignore`.
 Writing DocTests
 ----------------
 
+Tests start with for spaces!
+
 ```elm
 {-| returns the sum of two int.
 
     >>> add 41 1
-    === 42
+    42
```

Comment:
This is how doctest does it, but I'm actually not 100% sure on this. I originally considered mentioning dropping `===` to you, but decided I kinda liked that it was explicit rather than just magic. Maybe make a poll of the two styles on elm's slack or twitter?